maplibre_gl/lib/src/global.dart

```diff
@@ -83,6 +83,74 @@ Future<dynamic> deleteOfflineRegion(int id) => _globalChannel.invokeMethod(
       },
     );
 
+/// Creates a snapshot of a map with the specified parameters.
+/// 
+/// [width] and [height] define the dimensions of the snapshot in pixels.
+/// [styleUrl] is the URL of the map style to use.
+/// [cameraPosition] defines the camera position (center and zoom level) for the snapshot.
+/// [markers] is an optional list of markers to add to the snapshot.
+/// 
+/// Returns a [Uint8List] containing the PNG image data of the snapshot.
+Future<Uint8List> startMapSnapshot({
```

Comment:
nitpick: The function should not be named *start*MapSnapshot as it not only starts the process, but also waits for it to finish and returns the result.

How about `takeMapSnapshot` or `getMapSnapshot`?